internal/bus/chainsubscriber.go

```diff
@@ -236,20 +238,28 @@ func (s *chainSubscriber) applyChainUpdate(tx sql.ChainUpdateTx, cau chain.Apply
 	}
 
 	// new contracts - only consider the ones we are interested in
-	filtered := createdContracts[:0]
-	for _, fce := range createdContracts {
+	filtered := revisedContracts[:0]
+	for _, fce := range revisedContracts {
 		if s.isKnownContract(fce.ID) {
 			filtered = append(filtered, fce)
 		}
 	}
-	if err := tx.InsertFileContractElements(filtered); err != nil {
+	if err := tx.UpdateFileContractElements(filtered); err != nil {
 		return fmt.Errorf("failed to insert v2 file contract elements: %w", err)
 	}
 
 	// contract proofs
 	if err := tx.UpdateFileContractElementProofs(cau); err != nil {
 		return fmt.Errorf("failed to update file contract element proofs: %w", err)
 	}
+
+	// prune contracts 144 blocks after window_end
+	const fce_prune_window = 144
```

Comment:
inline const with underscores? Just checking, might be a new convention but looks off